### PR TITLE
update sample repo so that it can be successfully applied for 0.9

### PR DESCRIPTION
**Pull request by lilly-tecton**

### CODE REVIEW

File: ads/entities.py [hunk 1]
```diff
@@ -39,3 +39,5 @@
     owner='demo-user@tecton.ai',
     tags={'release': 'production'}
 )
+
+ad_embeddings_entity = Entity(name='ad_embeddings_entity', join_keys=["ad_id"], description='An ad used in the embeddings table', owner='demo-user@tecton.ai', tags={'release': 'production'})
```
Comment (lilly-tecton) [inline ads/entities.py:43]: `ad_id` is treated as a String in `ad_embeddings` but it's `Int64` in the other tables it references. thus, i create a new entity .

File: ads/features/stream_features/content_keyword_clicks_push.py [hunk 2]
```diff
@@ -19,5 +19,6 @@
     ttl=timedelta(days=30),
     tags={'release': 'production'},
     owner='demo-user@tecton.ai',
-    description='The ad clicks for a content keyword'
+    description='The ad clicks for a content keyword',
+    batch_trigger=BatchTriggerType.MANUAL
```
Comment (lilly-tecton) [inline ads/features/stream_features/content_keyword_clicks_push.py:23]: this was needed

File: repo.yaml
Comment (lilly-tecton) [inline repo.yaml:1]: was required

File: ads/entities.py (hunk 1 above)
Comment (jakednoble) [reply ads/entities.py:43]: @lilly-tecton , let's just chagne the Feature Table schema to an int64 and reuse the same entity

**Pull request merged**